mysql-test/suite/innodb/r/alter_charset.result

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE t (
+a CHAR(10) COLLATE latin1_general_ci PRIMARY KEY,
+b CHAR(10) COLLATE latin1_general_ci,
+c CHAR(10) COLLATE latin1_general_ci,
+UNIQUE KEY b_key(b)
+) ENGINE=INNODB;
+INSERT INTO t VALUES
+('aaa', 'aaa', 'aaa'), ('CCC', 'CCC', 'CCC'), ('bbb', 'bbb', 'bbb');
+ALTER TABLE t MODIFY a CHAR(10) COLLATE latin1_general_cs, ALGORITHM=INPLACE;
+ERROR 0A000: ALGORITHM=INPLACE is not supported. Reason: Cannot change column type INPLACE. Try ALGORITHM=COPY
```

Comment:
Repeating my request from https://github.com/MariaDB/server/pull/1251/files#r271254483

Please apply the following patch and adjust anything that you find with `git grep 'column type INPLACE'`:
```diff
diff --git a/sql/share/errmsg-utf8.txt b/sql/share/errmsg-utf8.txt
index 9f50aea1e22..e262ce62a5e 100644
--- a/sql/share/errmsg-utf8.txt
+++ b/sql/share/errmsg-utf8.txt
@@ -7035,7 +7035,7 @@ ER_ALTER_OPERATION_NOT_SUPPORTED_REASON_FK_RENAME
   eng "Columns participating in a foreign key are renamed"
 
 ER_ALTER_OPERATION_NOT_SUPPORTED_REASON_COLUMN_TYPE
-  eng "Cannot change column type INPLACE"
+  eng "Cannot change column type"
 
 ER_ALTER_OPERATION_NOT_SUPPORTED_REASON_FK_CHECK
   eng "Adding foreign keys needs foreign_key_checks=OFF"
```

Reply:
I think that should be done in a separate patch.

Reply:
Thanks, done in 4dc10ec68d354241768ce4779c024b4bb2ca897b and 7896503686477cce55d058d071d2858a8abf4b04.